### PR TITLE
replaced BaseException with Exception

### PR DESCRIPTION
**Pull request by sayantanb**
Replaced BaseException with Exception.

### CODE REVIEW

File: tnz/zti.py [hunk 1]
```diff
@@ -182,7 +182,7 @@ def cmdloop(self, intro=None):
                 self.__shell_mode()
                 print("\nKeyboardInterrupt")
 
-            except BaseException:
+            except Exception:
```
Comment (najohnsn) [inline tnz/zti.py:185]: This particular use of `BaseException` wants to remain `BaseException`. The intent here is to ensure that `postloop()` is called once `cmdloop()` may process something. For instance, if `SystemExit` is raised by `cmdloop()`, `postloop()` should still be called.

**Review by najohnsn — CHANGES_REQUESTED** [review]
Thanks for the contribution! One of the changes needs to not be made - I created a comment on the line. Please replace your commit with a _signed_ commit (i.e. `git -s`) that undoes that one line change, and I'll merge it in.

**Comment by najohnsn** [comment]
Forgot to point out that the [CONTRIBUTING](https://github.com/IBM/tnz/blob/main/CONTRIBUTING.md) document describes the commit signing.

**Comment by sayantanb** [comment]
Kept the BaseException on line 185 in zti.py and performed a signed commit.

**Review by najohnsn — APPROVED** [review]
LGTM

**Pull request merged**